src/components/Home/Home.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "../../components/Header/style.scss";
+
+export const Home = () => (
+  <div className="wrapper">
```

Comment:
you can't import styles from Header to Home component etc.

Reply:
Suspected that. But where should the styles be placed? 
Deleted the import and the wrapper class. It had just paddings in it.

Reply:
In the shared folder. Discussed on the daily

Reply:
done